scripts/test_differential_check.py

```
@@ -75,9 +75,20 @@ def test_calls_defined(self):
-        """Every call runs and returns something which can be fingerprinted."""
+        """Every call fingerprints, save the ones which exist to raise."""
+        # dump() records the error instead of the fingerprint when a call
+        # raises, so the comparison covers what each version says about a
+        # bad argument. Naming them here keeps a fourth from joining
+        # quietly, which would drop that call from the comparison.
+        raised = set()
         for name, call in get_calls().items():
-            assert digest(call()), f"{name} returned nothing"
+            try:
+                patch = call()
+            except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)